extensions/typescript-language-features/src/features/referencesCodeLens.ts

```diff
@@ -75,6 +82,15 @@ class TypeScriptReferencesCodeLensProvider extends TypeScriptBaseCodeLensProvide
 				}
 			// fallthrough
```

Comment:
This fall-through means that `showOnAllFunctions` will also apply to `let` and `var` symbols, which is not what we want. I suggest getting rid of the fallthrough

Reply:
Ah... Thank you 😖 How about this https://github.com/microsoft/vscode/pull/84689/commits/ec20ba205cee4b63e6162ae551d3ca9e19f53dda ?